UPGRADE.md

```diff
@@ -1,5 +1,9 @@
 # Upgrade to 2.14
 
+## Deprecated annotation mapping driver.
+
+You should switch to one of the other mapping drivers.
```

Comment:
This deprecation will affect a lot of projects out there. Let's try to be as helpful as possible.

```suggestion
Please switch to one of the other mapping drivers. Native attributes which PHP
supports since version 8.0 are probably your best option.
```